### PR TITLE
several enhancements to README

### PR DESCRIPTION
**Pull request by xsuchy**
and one fix in documentation (or confusion).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,16 +175,29 @@ specfile.set_version_and_release('2.1', release='3')
...
 #### Changelog
 
 ```python
-# adding a new entry, author is automatically determined
-# (using the same heuristics that rpmdev-packager uses) if possible
-specfile.add_changelog_entry('New upstream release 2.1')
+if not specfile.has_autochangelog:
+    # adding a new entry, author is automatically determined
+    # (using the same heuristics that rpmdev-packager uses) if possible
+    specfile.add_changelog_entry('- New upstream release 2.1')
```
Comment (nforro) [inline README.md:196]: `add_changelog_entry()` does nothing if `has_autochangelog` is `True` (documented [here](https://packit.dev/specfile/api/specfile#add_changelog_entry) in the docstring), so this isn't really necessary. But it doesn't hurt to have it here for clarity.

File: README.md [hunk 2]
```diff
@@ -175,16 +175,29 @@ specfile.set_version_and_release('2.1', release='3')
 specfile.set_version_and_release('2.1', preserve_macros=True)
 ```
 
+#### Bumping release
+
+To bump release and add a new changelog entry, we could use the following code:
```
Comment (nforro) [inline README.md:180]: Just to be consistent with the rest of the README:

```suggestion
To bump release and add a new changelog entry, you could use the following code:
```

**Review by nforro — APPROVED** [review]

**Comment by xsuchy** [comment]
Updated.

**Review by lbarcziova — APPROVED** [review]

**Comment by lbarcziova** [comment]
regate

**Pull request merged**